models/vista2d/scripts/workflow.py

```diff
@@ -0,0 +1,1205 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import gc
+import logging
+import os
+import shutil
+import sys
+import time
+from collections import OrderedDict
+from datetime import datetime
+
+import monai.transforms as mt
+import numpy as np
+import torch
+import torch.distributed as dist
+import yaml
+from monai.apps import get_logger
+from monai.auto3dseg.utils import datafold_read
+from monai.bundle import BundleWorkflow, ConfigParser
+from monai.config import print_config
+from monai.data import DataLoader, Dataset, decollate_batch
+from monai.metrics import CumulativeAverage
+from monai.utils import (
+    BundleProperty,
+    ImageMetaKey,
+    convert_to_dst_type,
+    ensure_tuple,
+    look_up_option,
+    optional_import,
+    set_determinism,
+)
+from torch.cuda.amp import GradScaler, autocast
+from torch.utils.data import WeightedRandomSampler
+from torch.utils.data.distributed import DistributedSampler
+from torch.utils.tensorboard import SummaryWriter
+
+mlflow, mlflow_is_imported = optional_import("mlflow")
+
+
+if __package__ in (None, ""):
+    from cell_distributed_weighted_sampler import DistributedWeightedSampler
+    from components import LabelsToFlows, LoadTiffd, LogitsToLabels
+    from utils import LOGGING_CONFIG, parsing_bundle_config  # type: ignore
+else:
+    from .cell_distributed_weighted_sampler import DistributedWeightedSampler
+    from .components import LabelsToFlows, LoadTiffd, LogitsToLabels
+    from .utils import LOGGING_CONFIG, parsing_bundle_config
+
+
+logger = get_logger("VistaCell")
```

Comment:
Hi @yiheng-wang-nv , can we change the `print` in this file to `logger.[info/error]`?

Reply:
Closing comments as discussed offline.